server/spec/controllers/cards.spec.js

```diff
@@ -1,28 +1,42 @@
-describe("Post request /api/v1/cards", () => {
-  const request = require("request")
-  const base_url = "http://localhost:5000/api/v1/cards"
+const request = require('request')
 
-  it("returns 201 code when card is created", (done) => {
+const endpoint = 'http://localhost:5000/api/v1/cards'
+
+describe('POST /api/v1/cards', () => {
+  it('returns 201 code when card is created', (done) => {
     request({
-      url: base_url,
-      method: "POST",
+      url: endpoint,
+      method: 'POST',
       json: true,
-      body: { description: "Test"},
-    }, (error, response, body) => {
+      body: { description: 'Test' },
+    }, (error, response) => {
       expect(response.statusCode).toEqual(201)
       done()
-    });
-  });
+    })
+  })
 
   it("returns 400 code when request doesnt't have description", (done) => {
     request({
-      url: base_url,
-      method: "POST",
+      url: endpoint,
+      method: 'POST',
       json: true,
       body: { },
-    }, (error, response, body) => {
+    }, (error, response) => {
       expect(response.statusCode).toEqual(400)
       done()
-    });
-  });
-});
+    })
+  })
+})
+
+describe('GET /api/v1/cards', () => {
+  it('returns 200 code when cards are loaded', (done) => {
+    request({
+      url: endpoint,
+      method: 'GET',
+      json: true,
+    }, (error, response) => {
+      expect(response.statusCode).toEqual(200)
+      done()
+    })
+  })
+})
```

Comment:
This file is removed in `master`. You may want to test via adding API docs for the new endpoint 😉